exporters/otlp/include/opentelemetry/exporters/otlp/otlp_grpc_exporter_options.h

```diff
@@ -6,13 +6,44 @@
 #include "opentelemetry/exporters/otlp/otlp_environment.h"
 
 #include <memory>
+// TODO: These requires C++17, is that OK? It seems like we need to support C++0x?
```

Comment:
We need support c++11. You can use absl::variant and absl::optional for alternative.

Reply:
Done.